client/my-sites/checkout/composite-checkout/components/additional-terms-of-service-in-cart.tsx

```diff
@@ -252,6 +252,137 @@ function getMessageForTermsOfServiceRecord(
 					},
 				}
 			);
+		case 'terms_for_bundled_trial_unknown_payment_method': {
+			if (
+				args.subscription_start_date &&
+				args.subscription_expiry_date &&
+				args.subscription_auto_renew_date
+			) {
+				if ( args.is_renewal_price_prorated ) {
+					const proratedRenewalArgs = {
+						args: {
+							endDate: moment( args.subscription_expiry_date ).format( 'll' ),
+							numberOfDays: args.subscription_pre_renew_reminder_days || 7,
+							productName: args.product_name,
+							regularPrice: args.regular_renewal_price,
+							renewalDate: moment( args.subscription_auto_renew_date ).format( 'll' ),
+							renewalPrice: args.renewal_price,
+							startDate: moment( args.subscription_start_date ).format( 'll' ),
+						},
+						components: {
+							manageSubscriptionlink: (
```

Comment:
There is a small typo here (don't forget to update translations as well):

```suggestion
							manageSubscriptionLink: (
```